public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -29,6 +30,11 @@ export const NotificationsStep = ({ alertUid }: NotificationsStepProps) => {
   const styles = useStyles2(getStyles);
 
   const [type] = watch(['type', 'labels', 'queries', 'condition', 'folder', 'name', 'manualRouting']);
+  const [showLabelsEditor, setShowLabelsEditor] = useState(false);
+
+  function onEditClick() {
```

Comment:
nit: Does it make sense to have a separate function for this?

Reply:
true, let's remove it